packages/app/src/cli/models/extensions/specifications/payments_app_extension.ts

```diff
@@ -24,6 +22,8 @@ const spec = createExtensionSpecification({
   deployConfig: async (config, _) => {
     if (config.targeting[0]!.target === 'payments.offsite.render') {
```

Comment:
> Is it 100% ensured that the array targeting has at least one element (and that this element has a target)?

It will, otherwise, it would fail the schema validation :)